### ISSUE TITLE
Invalid LaTeX ouput for line blocks with indented text.

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#535]
When using line blocks with indented text, LaTeX ouput is broken.

For instance, the following rst source code:

```
Some text.

| line1
|   line2
|     line3
| line4
|   line5
```

produces the following LaTeX code:

```
Some text.
{\raggedright{}line1\\
{\raggedright{}line2\\ 
{\raggedright{}line3}                            
line4\\                                                     
{\raggedright{}line5}
```
- LaTeX code is invalid (unbalanced braces).
- No new paragraph is started (//line1// is on the same line than //Some text.//).
- In the produced PDF (using pdflatex), //line2//, //line3// and //line5// are not indented.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/535
- Originally reported by: Anonymous
- Originally created at: 2010-10-12T17:13:06.479

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-10-22 10:10:51+00:00_

Thanks, fixed in <<cset 644e157fa5c1226d64916446ca03c8d36f28ae40>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:32+00:00_

Removing version: 0.6.6 (automated comment)
